docs/src/pages/LogoPage/LogoPage.js

```diff
@@ -0,0 +1,154 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { cssModules } from 'bpk-react-utils';
+import BpkImage from 'bpk-component-image';
+
+import getMarkdownString from '../../helpers/markdown-helper';
+import BpkMarkdownRenderer from '../../components/DocsPageBuilder/BpkMarkdownRenderer';
+import GuidelinesPageBuilder from '../../components/GuidelinesBuilder';
+import * as ROUTES from '../../constants/routes';
+import ContentGrid, {
+  ContentGridCard,
+} from '../../components/ContentGrid/ContentGrid';
+
+import STYLES from './logo-page.scss';
+import intro from './content/intro.md';
+
+const getClassName = cssModules(STYLES);
+
+const sections = [
+  {
+    id: 'intro',
+    content: <BpkMarkdownRenderer source={getMarkdownString(intro)} />,
+  },
+  {
+    id: 'symbol',
+    title: 'Our symbol',
+    backgroundStyle: 'light',
+    className: getClassName('bpk-docs-logo-page__image'),
+    content: (
+      <BpkImage
+        className={getClassName('bpk-docs-logo-page__image--constrained')}
+        altText="art direction"
+        width={758}
+        height={437}
+        src="https://js.skyscnr.com/sttc/bpk-content/our_symbol-af2f7219.png"
+      />
+    ),
+  },
+  {
+    id: 'wordmarque',
+    title: 'Our wordmarque',
+    className: getClassName('bpk-docs-logo-page__image'),
+    content: (
+      <BpkImage
+        className={getClassName(
+          'bpk-docs-logo-page__image--constrained',
+          'bpk-docs-logo-page__image--wordmarque',
+        )}
+        altText="art direction"
+        width={838}
+        height={190}
+        src="https://js.skyscnr.com/sttc/bpk-content/our_wordmarque-c8a774a8.svg"
+      />
+    ),
+  },
+  {
+    id: 'logo-suite',
+    title: 'Logo suite',
+    backgroundStyle: 'light',
+    content: (
+      <ContentGrid className={getClassName('bpk-docs-logo-page__grid')}>
+        <ContentGridCard className={getClassName('bpk-docs-logo-page__card')}>
+          <img
+            className={getClassName(
+              'bpk-docs-logo-page__suite',
+              'bpk-docs-logo-page__suite--vertical',
+            )}
+            src="https://js.skyscnr.com/sttc/bpk-content/logo_suite_vertical_logo-ee0c930e.svg"
```

Comment:
Could we create a const at the top of the file like `const LOGO_VERTICAL = "https://....svg"`?